{{ cookiecutter.app_name }}/{{ cookiecutter.project_name }}/settings.py

```diff
@@ -0,0 +1,146 @@
+"""
+Django settings for {{ cookiecutter.app_name }} pp. This is just a harness type
+project for testing and interacting with the app.
+
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+
+"""
+import os
+
+import dj_database_url
+import yaml
+
+CONFIG_PATHS = [
+    os.environ.get('{{ cookiecutter.project_name|upper }}_CONFIG', ''),
+    os.path.join(os.getcwd(), '{{ cookiecutter.project_name|lower }}.yml'),
+    os.path.join(os.path.expanduser('~'), '{{ cookiecutter.project_name|lower }}.yml'),
+    '/etc/{{ cookiecutter.project_name|lower }}.yml',
+]
+
+
+def load_fallback():
+    """Load optional yaml config"""
+    fallback_config = {}
+    config_file_path = None
+    for config_path in CONFIG_PATHS:
+        if os.path.isfile(config_path):
+            config_file_path = config_path
+            break
+    if config_file_path:
+        with open(config_file_path) as config_file:
+            fallback_config = yaml.safe_load(config_file)
+    return fallback_config
+
+FALLBACK_CONFIG = load_fallback()
+
+
+def get_var(name, default):
+    """Return the settings in a precedence way with default"""
+    return os.environ.get(name, FALLBACK_CONFIG.get(name, default))
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = get_var(
+    'SECRET_KEY',
+    '36boam8miiz0c22il@3&gputb=wrqr2plah=0#0a_bknw9(2^r'
```

Comment:
Since this is a template and variables are used in other places, is it easy to generate a random value here instead of hard-coded default? Many people never change their default anything.